lib/engine/game/base.rb

```diff
@@ -1367,11 +1369,12 @@ def discard_discount(train, price)
         (price * (100.0 - self.class::DISCARDED_TRAIN_DISCOUNT.to_f) / 100.0).ceil.to_i
       end
 
-      def end_game!(player_initiated: false)
+      def end_game!(game_end_reason)
         return if @finished
 
         @finished = true
-        @manually_ended = player_initiated
+        @game_end_reason = game_end_reason
+        @manually_ended = @game_end_reason == :manually_ended
```

Comment:
Do we need attr if the reason now states this? I don't think there are many places where we check manually_ended. ELO ratings is one...probably others.

Reply:
Okay, I'm good with migrating the db as a follow-up PR. I'd like to exclude those from ELO as well.

Reply:
Will be exclusion of game end results be defined per title?

Reply:
> Okay, I'm good with migrating the db as a follow-up PR. I'd like to exclude those from ELO as well.

I've created #12094 to track this.

> Will be exclusion of game end results be defined per title?

I wouldn't think this is necessary, going by the `game_end_reason` should be enough.